docs/source/kedro_project_setup/settings.md

```diff
@@ -0,0 +1,31 @@
+# Project settings
+
+A Kedro project's `settings.py` file contains the runtime configuration for the project, including registration of hooks and library components. This page explains how settings work and which settings are available.
+
+A project does not _need_ to have a settings file – in fact, by default, all code in `settings.py` is commented out. In the case that settings are not supplied, Kedro chooses sensible default values. You only need to edit `settings.py` if you wish to change to values other than the defaults.
```

Comment:
I just tried this and actually I was wrong - you do need a settings.py file. Sorry, my bad! Maybe we should change this so that we don't require a settings.py file though 🤔 
```suggestion
By default, all code in `settings.py` is commented out. In the case that settings are not supplied, Kedro chooses sensible default values. You only need to edit `settings.py` if you wish to change to values other than the defaults.
```